examples/web-chat/src/hooks.ts

```diff
@@ -210,3 +218,20 @@ export const usePeers = (params: UsePeersParams): UsePeersResults => {
 
   return peers;
 };
+
+// we only support websocket connections for now
+const isPeerDialable = async (peerId: PeerId, node: LightNode) => {
+  const peer = await node.libp2p.peerStore.get(peerId);
```

Comment:
I am wondering if we should do thins in `js-waku` and keep only `wss` peers in `peerStore` or make it configurable?